SpatialGDK/Source/SpatialGDKServices/Private/SpatialCommandUtils.cpp

```diff
@@ -7,6 +7,7 @@
 #include "Serialization/JsonSerializer.h"
 #include "SpatialGDKServicesConstants.h"
 #include "SpatialGDKServicesModule.h"
+#include "Windows/AllowWindowsPlatformTypes.h"
```

Comment:
Is this the right place for the header? I know it's alphabetical, but don't know if that's based on path initial letter or file name initial letter

Reply:
clang-format will place it in the correct place, I believe